.travis.yml

```diff
@@ -1,12 +1,30 @@
+dist: trusty
 sudo: false
 language: elm
+addons:
+  chrome: stable
+
+services:
+  - docker
 
 cache:
   directories: # so subsequent builds run faster
     - elm-stuff
     - $HOME/.elm
     - $HOME/.npm # https://stackoverflow.com/a/42523517/1148249
 
+before_install:
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+
 script:
   - elm-format --validate .
-  - npm run test:client
+  - npm run test:ci
+  - docker build -t "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" .
+  - docker tag "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" nakadi/nakadi-ui:latest
+
+deploy:
+  provider: script
+  script:
+    echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && bash docker push nakadi/nakadi-ui
```

Comment:
`bash docker push nakadi/nakadi-ui` why do you need `bash` here ?

Reply:
:man_facepalming: copy pasting